tools/clang/test/DXC/disable_paq.hlsl

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: dxil-1-8
 // RUN: %dxc -T lib_6_7 -disable-payload-qualifiers %s 2>&1 | FileCheck %s
 // RUN: %dxc -T lib_6_8 -disable-payload-qualifiers %s 2>&1 | FileCheck %s
```

Comment:
I think this change is fine, and this is just a side question based on seeing this test.  So feel free to resolve in order to merge the PR.

This only requires 1.8 due to the run line compiling to lib_6_8.  Is there a reason this target is needed?  As in, are there code changes that make a difference to this path between lib_6_7 and lib_6_8?

Reply:
It wants to test all the SM6.6+ versions.
So both lib_6_7 and lib_6_8 are added here.